news/2022-12-24-results-osu!taiko-featured-artist-cup-3.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title: "Results - osu!taiko Featured Artist Cup #3"
+date: 2022-11-24 10:00:00 +0000
+---
+
+The battle is over! Come join us in unveiling the winners of the osu!taiko mapping showdown of the year!
+
+![](https://assets.ppy.sh/contests/155/header.jpg)
+
+2 months ago, the [osu!taiko Featured Artist Cup](/wiki/Contests/o!tFAC) opened its gates to the world for the [third time](https://osu.ppy.sh/home/news/2022-10-23-osutaiko-featured-artist-cup-3), and after a careful process of judging, we're finally ready to crown the winners of this iteration!
+
+Before we proceed, we'd like to announce that because of the very high amount of entries, and the sheer quality of the submissions, we will be awarding the top 3 teams with badges, instead of only the winning team! Check out the [following page](/wiki/Contests/Monthly_Beatmapping_Contest#rewards) for a detailed list of the rewards.
+
+![](/wiki/Contests/o!tFAC/3/img/badge.png)
+
+## Results livestream
+
+In case you missed the results livestream, but still want to (somewhat) experience the thrill of revealing the top 21 entries, check out the full stream in the video below, commentated by [Hivie](https://osu.ppy.sh/users/14102976), [Axer](https://osu.ppy.sh/users/7299864), [Greenshell](https://osu.ppy.sh/users/8693851), and [overdahedge2014](https://osu.ppy.sh/users/9864847)!
+
+<div align="center">
+    <iframe width="95%" height="400" src="https://player.twitch.tv/?parent=osu.ppy.sh&video=v1687153575" frameborder="0" allowfullscreen></iframe>
+</div>
+
+## Results summary
+
+### 3rd place
+
+Don't let the cute name fool you, because team **:3** didn't pull any punches in their take of [Camellia - BAD ACCESS (FROM A MOE MAID)](https://osu.ppy.sh/beatmapsets/1908855), with [woosungko](https://osu.ppy.sh/users/14184157), the winner of the first Featured Artist Cup, and [Horiiizon](https://osu.ppy.sh/users/8071438) giving it their all to create a very enjoyable and memorable experience, which has earned them a handsome 3rd place finish!
```

Comment:
```suggestion
Don't let the cute name fool you, because team **:3** didn't pull any punches in their take of [Camellia - BAD ACCESS (FROM A MOE MAID)](https://osu.ppy.sh/beatmapsets/1908855), with [woosungko](https://osu.ppy.sh/users/14184157), the winner of the first Featured Artist Cup, and [Horiiizon](https://osu.ppy.sh/users/8071438) giving it their all to create a very enjoyable and memorable experience, which has earned them a handsome 3rd-place finish!
```